package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "next-experiments",
+  "version": "1.0.4-alpha",
+  "description": "Infrastructure for running ABTests with next.js",
+  "main": "index.js",
```

Comment:
Have you tried using `dist/index.js`? It may let you skip the `postinstall` / `bin/install` script entirely

Reply:
Fixed